Makefile

```diff
@@ -68,13 +68,18 @@ flake8:
 	$(PYTHON) -m flake8 $(CHECK_DIRS) --exclude gettext38.py,contextlib38.py
 
 unit: gitdeps
-	python3 -m unittest discover
+	python3 -m pytest --ignore curtin --ignore probert \
+		--ignore subiquity/tests/api
+
+api:
+	@[ "$(shell lsb_release -sr | cut -d. -f1)" -le "18" ] || \
```

Comment:
So I don't think we need this any more, we should just stop testing on bionic.

Reply:
Agree, fixed.